src/renderers/shaders/ShaderLib/meshphysical_frag.glsl

```diff
@@ -19,6 +19,10 @@ varying vec3 vViewPosition;
 
 #endif
 
+#ifdef USE_SPHERICAL_HARMONICS
+	varying vec3 worldNormal;
```

Comment:
Unnecessary, look at how we are already query the hemiphere light and the environment map:

https://github.com/atteneder/three.js/blob/6606a8b35f26bb7ecd230a111423996e82551a37/src/renderers/shaders/ShaderChunk/lights_template.glsl#L124

Notice it passes in geometry structure?